tinytuya/core/__init__.py

```diff
@@ -16,3 +16,9 @@
 from .core import *
 from .core import __version__
 from .core import __author__
+
+# Conditionally import async modules, only available in Python 3.5 and above.
+import sys
+if sys.version_info >= (3, 5):
+    from .XenonDeviceAsync import *
+    from .DeviceAsync import *
```

Comment:
Yeah, I know... basically anything less than 3.5 is going to NOP. I need to add an error and assert python 3.5+ support requirements.